composer.json

```diff
@@ -32,7 +32,8 @@
         "ext-libxml": "*",
         "ext-mbstring": "*",
         "ext-openssl": "*",
-        "ext-PDO": "*",
+        "ext-pdo": "*",
+        "ext-pdo_sqlite": "*",
```

Comment:
This shouldn't be added here, instead the relevant recommend package entry should be updated to include this feature as requiring this extension.

Reply:
Sorry, I don't follow?

Reply:
Adding pdo sqlite as a required php extension breaks the library for installs without that extension. Instead, we should just add to the description that we already have for this dependencies entry in the recommended dependencies package.

Reply:
Given this will be core functionality though, won't it actually be a dependency going forward? We couldn't just leave it as a suggestion.

Reply:
Unless a high traffic part of the backend or cms uses it then no. Technically speaking supporting sqlite is already core functionality, and look at how the assetic library handles this. If you want to enable this specific functionality then you need to provide the related dependency.